### PR TITLE
Review module

### PR DESCRIPTION
**Pull request by matthewb2022**

### CODE REVIEW

File: src/components/BeachPage/BeachPage.js [hunk 1]
```diff
@@ -17,7 +19,7 @@ const BackImage = (props) => {
                             fractions={2}
                             emptySymbol={(<Star className={styles.starempty}></Star>)}
                             fullSymbol={(<Star className={styles.starfull}></Star>)}
-                            initialRating={props.beach.rating}
+                            initialRating={props.beach.ratings.average}
```
Comment (mjmayank1) [inline src/components/BeachPage/BeachPage.js:22]: let's undo this for now

**Review by mjmayank1 — CHANGES_REQUESTED** [review]
looks good to me, just the one change about rating average

**Comment by matthewb2022** [comment]
Fixed based on comment

**Comment by mjmayank1** [comment]
it also looks like theres a merge conflict in server.js that you need to fix before this can be merged

**Review by mjmayank1 — APPROVED** [review]

**Pull request merged**